index.html

```diff
@@ -50,6 +50,146 @@ <h3 class="helpingtext">LET'S CONNECT</h3>
+            <div class="text-title">
+                <a href="#" class="client">CANOPY</a>
+                <img class="dot" src="req/dot.png" alt="dot">
+                <a href="#" class="client-skills">BACK END DEV</a>
+                <img class="dot" src="req/dot.png" alt="dot">
+                <a href="#" class="period">2015</a>
+            </div>
+            <p class="content-text">A daily selection Of privately personalized reads; no accounts or sign-ups required</p>
+            <ul class="tags">
+                <li class="style">html</li>
+                <li class="style">css</li>
+                <li class="style">javascript</li>
+            </ul>
+            <a href="#" class="link">See Project</a>
+        </div>
+    </article>  
+</section>
+<section class="About-container">
+    <div class="top">
+        <h2 class="about-head">About </br> Myself</h2>
+        <p class="text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h3 class="helpingtext">LET'S CONNECT</h3>
+        <div class="belowtext">
+            <ul >
+                <li>
+                    <a href="#"><img src="req/twitter.png" alt="twitter"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="req/Linkedin.png" alt="linkedin"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="req/M.png" alt="M"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="req/Github.png" alt="Github"></a>
+                </li>
+                <li>
+                    <a href="#"><img src="req/V.png" alt="V"></a>
+                </li>
+            </ul>
+        </div>
+        <a href="#" class="link">Get my resume</a>
+    </div>
+    <div class="about-skills">
+        <div class="lang-detail">
+        <h3 class="lang-head">Languages</h3>
+        <i class="arrow down"></i>
+        </div>
+        <ul class="lang-names">
+            <li class="lang-container">
+                <img class="lang-img" src="req/js.png" alt="javascript">
+                <p class="lang">javascript</p>
+            </li>
+            <li class="lang-container">
+                <img class="lang-img" src="req/html.png" alt="html">
+                <p class="lang">HTML</p>
+            </li>
+            <li class="lang-container">
+                <img class="lang-img" src="req/css.png" alt="css">
+                <p class="lang">CSS</p>
+            </li>
+        </ul>
+        <div class="lang-detail bottom-line">
+            <h3 class="Lang-head">Frameworks</h3>
+            <i class="arrow right"></i> 
+        </div>
+        <div class="lang-detail bottom-line">
+            <h3 class="Lang-head">Skills</h3>
+            <i class="arrow right"></i>    
+        </div>
```

Comment:
-  [ ] As requested by the previous reviewer, kindly use a nested unordered list for your list of technologies. Your list of technologies should look like this;
```html
<ul>
  <li>Languages
    <ul>
      <li>HTML</li>
      <li>CSS</li>
      <li>Javascript</li>
    </ul>
  </li>
  <li>Frameworks</li>
  <li>Skills</li>
</ul>
```